heimdall-middleware-spec/src/test/java/br/com/conductor/heimdall/middleware/util/helpermock/call/RequestMockTest.java

```diff
@@ -0,0 +1,50 @@
+package br.com.conductor.heimdall.middleware.util.helpermock.call;
+
+import org.junit.Test;
+import java.util.HashMap;
+
+import static org.junit.Assert.*;
+
+public class RequestMockTest {
+
+  @Test
+  public void testHeader() {
+    final RequestMock requestMock = new RequestMock();
+    assertEquals(new HashMap<String, String>(), requestMock.header().getAll());
+  }
+
+  @Test
+  public void testQuery() {
+    final RequestMock requestMock = new RequestMock();
+    assertEquals(new HashMap<String, String>(), requestMock.query().getAll());
+  }
+
+  @Test
+  public void testSetAppNameInput() {
+    final RequestMock requestMock = new RequestMock();
+    requestMock.setAppName("testName");
+    assertEquals("testName", requestMock.getAppName());
+  }
+
+  @Test
+  public void testSetBodyInput() {
+    final RequestMock requestMock = new RequestMock();
+    requestMock.setBody("testBody");
+    assertEquals("testBody", requestMock.getBody());
+  }
+
+  @Test
+  public void testSetUrl() {
+    final RequestMock requestMock = new RequestMock();
+    requestMock.setUrl("www.google.com");
+    assertEquals("www.google.com", requestMock.getUrl());
+  }
+
+  @Test
+  public void testSetPathParam() {
+    final RequestMock requestMock = new RequestMock();
+    assertEquals(null, requestMock.pathParam(""));
```

Comment:
Apologies, I think I missed this earlier: use assertNull(foo) instead of assertEquals(null, foo)
(similarly with assertTrue and assertFalse() if the situation arises)